<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationWidget</class>
 <widget class="QWidget" name="NavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(91, 91, 91, 255));</string>
  </property>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/Images/Home.png</normaloff>:/Images/Home.png</iconset>
   </property>
  </widget>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>941</width>
     <height>621</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Navigation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/Images/icon_navigation.png</normaloff>:/Images/icon_navigation.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>MENÃœ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Media">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/Images/music_icon_2.png</normaloff>:/Images/music_icon_2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Phone">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/Images/phone_icon_1.png</normaloff>:/Images/phone_icon_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Radio">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/Images/radio_icon_1.png</normaloff>:/Images/radio_icon_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
